docs/design-documents/20221024-stream-inspector.md

```diff
@@ -0,0 +1,143 @@
+# Stream Inspector
+
```

Comment:
Need to add a note that some inspector data may be dropped (e.g. if inspector is too slow).